src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/ClaimantContentProvider.java

```diff
@@ -40,7 +46,16 @@ public ClaimantContent createContent(Party claimant, String submitterEmail) {
             personContent.getCorrespondenceAddress(),
             submitterEmail,
             personContent.getContactPerson(),
-            personContent.getBusinessName()
+            personContent.getBusinessName(),
+            personContent.getMobileNumber(),
+            personContent.getDateOfBirth()
         );
     }
+
+    private Optional<LocalDate> claimantDateOfBirth(Party party) {
```

Comment:
Please move it into `uk.gov.hmcts.cmc.domain.utils.PartyUtils`. That is the place we have similar logic.

Reply:
Attached the latest claim pdf and sealed copy
[000MC047-claim-receipt.pdf](https://github.com/hmcts/cmc-claim-store/files/1826424/000MC047-claim-receipt.pdf)
[000MC047-sealed-claim.pdf](https://github.com/hmcts/cmc-claim-store/files/1826425/000MC047-sealed-claim.pdf)